src/components/SideNav/SideNav.story.scss

```diff
@@ -0,0 +1,36 @@
+@import '../../globals/vars';
+
+.storybook-container {
+  padding: 0 !important;
+}
+
+.#{$iot-prefix}--main-content {
```

Comment:
Do we need to document these styles somewhere to explain how/why we use them for the story? Code comments with rationale would be helpful for this file

Reply:
Or should we have two stories for sidenav?
```
▼ Sidenav
 ┣ without push
 ┣ with push
```

Reply:
I have documented the use of "pushing" the content in the info section.